### ISSUE TITLE
SystemStackError: stack level too deep

### ISSUE THREAD

**Issue by alexey-klimuk** [hanami/model#476]
Hi guys,
I have some issues calling `update` method for repo.

`
repository.update(id, attrs)
`

Backtrace
```
1 File "/app/vendor/bundle/ruby/2.4.0/gems/hanami-model-1.0.0/lib/hanami/model/plugins/schema.rb" line 26 in []
2 File "/app/vendor/bundle/ruby/2.4.0/gems/hanami-model-1.0.0/lib/hanami/model/plugins/timestamps.rb" line 36 in []
3 File "/app/vendor/bundle/ruby/2.4.0/gems/hanami-model-1.0.0/lib/hanami/model/plugins/mapping.rb" line 26 in []
4 File "/app/vendor/bundle/ruby/2.4.0/gems/hanami-model-1.0.0/lib/hanami/model/plugins/schema.rb" line 26 in []
5 File "/app/vendor/bundle/ruby/2.4.0/gems/hanami-model-1.0.0/lib/hanami/model/plugins/timestamps.rb" line 36 in []

....

244 File "/app/vendor/bundle/ruby/2.4.0/gems/hanami-model-1.0.0/lib/hanami/model/plugins/timestamps.rb" line 36 in []
245 File "/app/vendor/bundle/ruby/2.4.0/gems/hanami-model-1.0.0/lib/hanami/model/plugins/mapping.rb" line 26 in []
246 File "/app/vendor/bundle/ruby/2.4.0/gems/hanami-model-1.0.0/lib/hanami/model/plugins/schema.rb" line 26 in []
247 File "/app/vendor/bundle/ruby/2.4.0/gems/rom-sql-1.3.2/lib/rom/sql/commands/update.rb" line 27 in execute
248 File "/app/vendor/bundle/ruby/2.4.0/gems/rom-3.2.2/lib/rom/command.rb" line 279 in call
249 File "/app/vendor/bundle/ruby/2.4.0/gems/rom-sql-1.3.2/lib/rom/sql/commands/error_wrapper.rb" line 12 in call
250 File "/app/vendor/bundle/ruby/2.4.0/gems/rom-3.2.2/lib/rom/commands/composite.rb" line 17 in call
251 File "/app/vendor/bundle/ruby/2.4.0/gems/rom-repository-1.3.2/lib/rom/repository/class_interface.rb" line 133 in block (2 levels) in define_restricted_command_method
252 File "/app/vendor/bundle/ruby/2.4.0/gems/hanami-model-1.0.0/lib/hanami/repository.rb" line 353 in update
253 File "/app/apps/api/actions/commands/consume_deep_social_refreshed_data.rb" line 61 in update_influencer_platform
254 File "/app/apps/api/actions/commands/consume_deep_social_refreshed_data.rb" line 24 in block (2 levels) in call
255 File "/app/vendor/bundle/ruby/2.4.0/gems/rom-sql-1.3.2/lib/rom/sql/transaction.rb" line 13 in block in run
256 File "/app/vendor/bundle/ruby/2.4.0/gems/sequel-4.46.0/lib/sequel/database/transactions.rb" line 224 in _transaction
257 File "/app/vendor/bundle/ruby/2.4.0/gems/sequel-4.46.0/lib/sequel/database/transactions.rb" line 199 in block in transaction
258 File "/app/vendor/bundle/ruby/2.4.0/gems/sequel-4.46.0/lib/sequel/database/connecting.rb" line 301 in block in synchronize
259 File "/app/vendor/bundle/ruby/2.4.0/gems/sequel-4.46.0/lib/sequel/connection_pool/threaded.rb" line 107 in hold
260 File "/app/vendor/bundle/ruby/2.4.0/gems/sequel-4.46.0/lib/sequel/database/connecting.rb" line 301 in synchronize
261 File "/app/vendor/bundle/ruby/2.4.0/gems/sequel-4.46.0/lib/sequel/database/transactions.rb" line 165 in transaction
262 File "/app/vendor/bundle/ruby/2.4.0/gems/rom-sql-1.3.2/lib/rom/sql/transaction.rb" line 13 in run
263 File "/app/vendor/bundle/ruby/2.4.0/gems/rom-3.2.2/lib/rom/gateway.rb" line 186 in transaction
264 File "/app/vendor/bundle/ruby/2.4.0/gems/rom-repository-1.3.2/lib/rom/repository.rb" line 288 in transaction
265 File "/app/apps/api/actions/commands/consume_deep_social_refreshed_data.rb" line 23 in block in call
266 File "/app/apps/api/actions/commands/consume_deep_social_refreshed_data.rb" line 5 in each
267 File "/app/apps/api/actions/commands/consume_deep_social_refreshed_data.rb" line 5 in call
```
Details:
`consume_deep_social_refreshed_data.rb" line 60-62` 
```
def update_something(id, attrs)
  repository.update(id, attrs)
end
```
`consume_deep_social_refreshed_data.rb" line 5-25`
```
items.each do |item|
  repository.transaction do
    update_something(item.id, item.attrs)
    ...
  end
end
```

Any ideas why it can happen?

**Comment by flash-gordon** [comment]
Hello, this was fixed in 1.0.1 https://github.com/hanami/model/blob/563b1d58ceefca1ea6136044e137b2970dbed7ee/CHANGELOG.md#fixed-6

**Comment by alexey-klimuk** [comment]
@flash-gordon thanks, I've missed it for some reasons when was reading changelog 👍
